sqlglot/dialects/clickhouse.py

```diff
@@ -743,7 +743,6 @@ def _parse_function(
                     "expressions": anon_func.expressions,
                 }
                 if parts[1]:
-                    kwargs["parts"] = parts
```

Comment:
This isn't used after parsing, it's only used here to determine the type of the produced node.